server/plugin_test.go

```diff
@@ -423,67 +423,38 @@ func TestSyncUsers(t *testing.T) {
 	}
 }
 
-func TestConnectTeamsAppClient(t *testing.T) {
```

Comment:
This wasn't really testing anything other than calling order, and there's no way to mock it now that the implementation is simpler. I'll revisit this when we get to the integration testing epic.